ideas/142-wallet-compatibility/README.md

```diff
@@ -0,0 +1,111 @@
+---
+id: 142-wallet-compatibility
+title: Wallet Compatibility
+status: Active
+created: 2018-04-04
+category: core
+lead-contributor: pilu
+contributors:
+    - pilu
+    - antdanchenko
+    - asemiankevich
+    - flexsurfer
+    - chadyj
+    - jpbowen
+exit-criteria: yes
+success-metrics: yes
+clear-roles: yes
+future-iterations: yes
+roles-needed:
+---
+
+## Meta
+
+    Idea: #142
+    Title: Wallet Compatibility
+    Status: Active
+    Created: 2018-04-04
+    Replaces (*optional): #94
+
+## Summary
+
+Users should be able to use `dapps` on their browser and continue to use them with the same account in Status, and vice versa.
+
+## Swarm Participants
+
+- Lead Contributor: @pilu
+- Testing & Evaluation: @antdanchenko
+- Testing & Evaluation: @asemiankevich
+- Contributor: @flexsurfer
+- Contributor: <!-- @username -->
+- PM: @chadyj
+- UX (if relevant): @jpbowen
+<!-- - Contributor: @username -->
+
+## Product Overview
+
+Other wallets like MetaMask, Toshi, Cipher, etc., implements BIPs 39/32/44 like Status, and so they derive the keys starting from 12 mnemonic words.
+
+Given that they use the same standards, users can start playing with a `dapp` with one of those apps, and continue with another one without problems. They can open one of the other apps, and use the same 12 mnemonic words to import their existing account (private key).
+
+In Status, we implement the same standards to derive the keys starting from the 12 mnemonic words but using different constants, and we use the password added as an extra entropy in the generation of the seed.
+
+This means that users can click on "Add existing account" and successfully "import" an account with the 12 mnemonic words that they generated in a different wallet software, but ending up with a different key and address.
+Since the generation of the key is different, the account and address are not the ones that the user expected to import.
```

Comment:
Can you briefly mention how to solve this? Is this a major problem?

Reply:
@chadyj in the Technical overview section, I describe the changes that fix it